stock_picking_show_return/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Show returns on stock pickings',
+    'version': '10.0.1.0',
```

Comment:
Sorry, I should be clearer : 10.0.1.0.0

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers